airflow/providers/cncf/kubernetes/CHANGELOG.rst

```diff
@@ -44,6 +44,7 @@ Bug Fixes
 * ``Fix KubernetesPodOperator duplicating logs when interrupted (#33500)``
 * ``Fix 2.7.0 db migration job errors (#33652)``
 * ``Inspect container state rather than last_state when deciding whether to skip (#33702)``
+* ``Update pod.py (#33779)``
```

Comment:
FYI in providers we can not update commit messages in release notes

Reply:
@eladkal -> yes, we can update the message in CHANGELOG - we cannot update them in COMMITS. The only thing important is to make sure we leave `(#33779)`.

Reply:
I tried that once and it didn't work(?) let me try again

Reply:
It was proabaly in "COMMITS" :). The COMMITS list is always regenerated from scratch when the docs are generated - just to make sure it is accurate. The CHANGELOG on the other hand is only updated with new things (based on the 'PR#') and we only add new things to changelog if we see the #PR  number is missing there. This allows to manually modify and move around the individual entries - hide them, reword them, etc.

Reply:
Also this allows to make "iterations" of release. Whenever we "prepare docs" - like this one from the new main, only the "new" things that had not been present the last time in the CHANGELOG are added. That's why when you cancel an RC and prepare the new one, all those "new entries" land in the "commented" section stating "Please update and move the new entries that appeared since the last "prepare documantation" - that effectively allows to prepare same version several times (several RC candidates or even several times before deciding on RC). The (PR#) is the key to see if new entries should be added when you generate the docs from new main.